services/ai-service/.env

```
@@ -0,0 +1,23 @@
+# Server
+NODE_ENV=development
+PORT=3002
+
+# Gemini AI
+GEMINI_API_KEY=AIzaSyB0VMfHasPKVNMMrdEcbdQeJNSF0IZeERQ
+GEMINI_MODEL=gemini-2.5-flash
+# Database
+DB_HOST=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_HOST key should go before the GEMINI_API_KEY key

(UnorderedKey)